server/Dockerfile

```diff
@@ -37,8 +48,11 @@ USER 1000
 COPY docker-entrypoint.sh /usr/local/bin
 COPY is_healthy.sh /usr/local/bin
 COPY is_running.sh /usr/local/bin
+# S2I Scripts
+COPY .s2i /usr/local/s2i
 
-ENTRYPOINT ["docker-entrypoint.sh"]
+# OpenShift S2I requires to use CMD not ENTRYPOINT here
+CMD ["docker-entrypoint.sh"]
```

Comment:
Unfortunately they had to revert it: https://github.com/openshift/source-to-image/issues/475#issuecomment-215891632.

Let me try to work it around. Maybe I will come up with some idea.